python/lsst/ts/scriptqueue/utils.py

```diff
@@ -1,6 +1,15 @@
-__all__ = ["find_public_scripts"]
+__all__ = ["find_public_scripts", "configure_logging", "generate_logfile", "DETAIL_LEVEL"]
 
 import os
+import logging
+import time
+
+DETAIL_LEVEL = {
```

Comment:
I'm very uneasy about you remapping 0-3 to 10-40. I think it would be much nicer to just use the standard values -- which you can list in the help for that argument.

Or if you feel like being fancy, support integers AND names, e.g. `--level=30` or `--level=info`, in which case you will want a dict of name: value instead of int: value (and code that can tell the difference between a string and an int, e.g. something like this (untested):
```
LOG_LEVEL_DICT = dict(
    error=logging.ERROR,
    warning=logging.WARNING,
    info=logging.INFO,
    debug=logging.DEBUG,
)

...

try:
    level = int(level_str)
except ValueError:
    level = LOG_LEVEL_DICT[level_str)]
except KeyError:
    level_names = ", ".join(LOG_LEVEL_DICT.keys()
    raise ValueError(f"{level_str} must be an integer or one of: {level_names}")
```